src/workspaces/features/RunPipelineDialog/ParameterField.tsx

```diff
@@ -60,6 +60,19 @@ const ParameterField = (props: ParameterFieldProps) => {
       />
     );
   }
+  if (parameter.multiple) {
+    return (
+      <Textarea
```

Comment:
Could you add an explanation on how to use the textarea to the user below the field ?